src/api/FileCollaborators.js

```diff
@@ -58,11 +72,13 @@ class FileCollaborators extends MarkerBasedAPI {
      */
     getFileCollaborators(
         id: string,
-        successCallback: Function,
+        successCallback: ({ entries: Array<SelectorItem<UserMini | GroupMini>>, next_marker: ?string }) => void,
```

Comment:
`SelectorItems<UserMini | GroupMini>` or stop using the `SelectorItems` type altogether?